addons/website_crm_partner_assign/static/src/js/crm_partner_assign.js

```diff
@@ -286,4 +287,15 @@ publicWidget.registry.crmPartnerAssign = publicWidget.Widget.extend({
         }
     },
 });
+
+if ($("#force_opportunities_display").length !== 0 ) {
+    publicWidget.registry.PortalHomeCounters.include({
+        /**
+         * @override
+         */
+        _getCountersAlwaysDisplayed() {
+            return this._super(...arguments).concat(['opp_count']);
+        }
+    });
+}
```

Comment:
Not super confident about having the condition at the root of the file. Could it maybe not be loaded or something? Globally just feels a bit wrong to conditionally "override".

Would rather have the condition inside the `_getCounters` directly if possible